### PR TITLE
ci: Update PR template to simplify (no-changelog)

### PR DESCRIPTION
**Pull request by mutdmour**
Update PR template to simplify and reduce noise.

N/A

## Review / Merge checklist
- [x] PR title and summary are descriptive. **Remember, the title automatically goes into the changelog. Use `(no-changelog)` otherwise.** ([conventions](https://github.com/n8n-io/n8n/blob/master/.github/pull_request_title_conventions.md))
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [ ] Tests included.
   > A bug is not considered fixed, unless a test is added to prevent it from happening again.
   > A feature is not complete without tests.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,23 +1,16 @@
 ## Summary
-Provide details about your pull request and what it adds, fixes, or changes. Photos and videos are recommended.
+> Describe what the PR does. And how to test. Photos and videos are recommended.
```
Comment (tomi) [inline .github/pull_request_template.md:2]: Should we also mention that you should remove sections that are not relevant?

**Review by tomi — DISMISSED** [review]
Left couple comments but LGTM 👍

File: .github/pull_request_template.md (hunk 1 above)
Comment (mutdmour) [reply .github/pull_request_template.md]: all the sections are relevant now.. tickets should almost always be there. description is always needed. checklist is okay if it's there but not fully completed.. 
I am okay with default text making it into normal PRs.. I don't expect engineers to clean up the description every time.

**Review by tomi — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.21.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.21.0)

**Pull request merged**